lib/config/presets/index.spec.ts

```diff
@@ -446,6 +446,17 @@ describe('config/presets/index', () => {
         presetSource: 'local',
       });
     });
+    it('parses local with spaces and subdirectory', () => {
```

Comment:
The test only checks names with one space though.

Should we add another test that checks for two spaces?
Maybe something like this:

```
    it('parses local with 2 spaces and subdirectory', () => {
      expect(
        presets.parsePreset('local>A2B C D/A2B_Renovate//some-dir/some-file')
      ).toEqual({
        repo: 'A2B C D/A2B_Renovate',
        params: undefined,
        presetName: 'some-file',
        presetPath: 'some-dir',
        presetSource: 'local',
      });
    });
```

Reply:
I must admit that I just replicated the work from https://github.com/renovatebot/renovate/commit/55d3c0468dc6ef76267e5bc9eaab548f0c5e7283
And I can see from the regex that multiple spaces will work, because of the 1 or more operator.
If that wasn't there, all tests regarding the preset parsing will fail anyway.